libs/payments/ui/src/lib/server/components/PurchaseDetails/index.tsx

```diff
@@ -189,7 +202,39 @@ export async function PurchaseDetails(props: PurchaseDetailsProps) {
           </span>
         </li>
       </ul>
-      {/* TODO - Add InfoBox as part of Coupon Form - Consider adding as child component */}
+
+      {!discountType || discountType === 'forever' ? null : discountEnd &&
+        discountEnd > planRenewalDate &&
+        discountType === 'repeating' ? (
+        <div className="flex items-center justify-center gap-2 text-green-900 pt-2 pb-6 font-medium">
+          <Image src={infoLogo} alt="" />
```

Comment:
We had considered this image decorative as it does not convey important information/value for the customer to understand the content of the message. As it could confuse and/or clutter the customer's experience when using a screen reader, decorative images don't require an alt text.